src/graphql/search/cms/index.ts

```diff
@@ -26,21 +29,30 @@ const cmsSearch = (type: string) => async (
     types: [type],
     filters: filterInput,
     sort,
+    subType,
   })
 
-  const cmsFilters: any = await loaders.cms.load(
+  const formattedResults = getFormattedResults(results)
+
+  const cmsThemeFilters: any = await loaders.cms.load(
```

Comment:
Can we use another type other than `any` here? Perhaps like `subTypeFilters` below here.